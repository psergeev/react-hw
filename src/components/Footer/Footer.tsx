import * as React from 'react';
import './Footer.scss';

export default React.memo(() => (
    <footer>netflixroulette</footer>
));
