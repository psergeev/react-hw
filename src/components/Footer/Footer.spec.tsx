import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Footer from '.';

test('Footer component works as expected', () => {
    const component = renderer.create(
        <Footer />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
