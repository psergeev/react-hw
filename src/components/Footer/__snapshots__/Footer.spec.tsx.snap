// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component works as expected 1`] = `
<footer>
  netflixroulette
</footer>
`;
